### PR TITLE
update copy on pricing page

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by Hasmin-AC — APPROVED** [review]

**Comment by Hasmin-AC** [comment]
now that there isn't too much text for contact us, @Yndira-E what do you think of making this a heading with the Form below it instead of on the side?

**Comment by Yndira-E** [comment]
Hi @Hasmin-AC this can be part of the design review I'll be doing soon, pricing page will be the first one 😉. So feel free to merge as is and we'll handle that in a different PR.

File: src/pricing/index.njk [hunk 1]
```diff
@@ -580,10 +580,10 @@ hubspot:
...
-    setupPricingSlider("#team-monthly-slider", 300, 5, "#team-monthly-price-int", "#team-monthly-price-dec", "#team-monthly-price-note", "#team-monthly-value", false);
-    setupPricingSlider("#enterprise-monthly-slider", 600, 10, "#enterprise-monthly-price-int", "#enterprise-monthly-price-dec", "#enterprise-monthly-price-note", "#enterprise-monthly-value", false);
+    setupPricingSlider("#team-annual-slider", 250, 3, "#team-annual-price-int", "#team-annual-price-dec", "#team-annual-price-note", "#team-annual-value", true);
+    setupPricingSlider("#enterprise-annual-slider", 500, 5, "#enterprise-annual-price-int", "#enterprise-annual-price-dec", "#enterprise-annual-price-note", "#enterprise-annual-value", true);
+    setupPricingSlider("#team-monthly-slider", 300, 3, "#team-monthly-price-int", "#team-monthly-price-dec", "#team-monthly-price-note", "#team-monthly-value", false);
+    setupPricingSlider("#enterprise-monthly-slider", 600, 5, "#enterprise-monthly-price-int", "#enterprise-monthly-price-dec", "#enterprise-monthly-price-note", "#enterprise-monthly-value", false);
```
Comment (Yndira-E) [inline src/pricing/index.njk:586]: I can see new minimum instances. There are other sections of the code that need to be updated as well, I'll update what's needed (maybe I'll iterate on the script instead)

**Review by Yndira-E — CHANGES_REQUESTED** [review]

**Review by Yndira-E — APPROVED** [review]

File: src/pricing/index.njk (hunk 1 above)
Comment (Yndira-E) [reply src/pricing/index.njk:589]: I already updated the script

**Comment by Yndira-E** [comment]
Since I'm the last one approving, I'm merging it now.

**Pull request merged**